packages/vats/src/proposals/upgrade-paRegistry-proposal.js

```diff
@@ -51,6 +54,9 @@ export const getManifestForUpgradingRegistry = (_powers, { registryRef }) => ({
       brand: { consume: { [Stable.symbol]: par } },
     },
   },
+  installations: {
+    walletFactory: restoreRef(registryRef),
```

Comment:
`walletFactory` looks like a copy-and-paste-o

These changes don't seem to get tested.

Reply:
I can think of a contrived way to test for this particular problem:

 1. run some/all of these upgrades
 2. for each entry in `agoricNames.installation`, get the bundleId from the installation; extract the bundle from the kernel db; get the entry module of the bundle
 3. check that the entry module makes sense compared to the key of `agoricNames.installation`

But it seems like a lot of work compared to the gain. Especially since the harm from this goofy code is minimal: we don't intend to start another walletFactory. `z:acceptance` tests all the things we _do_ plan to do (or: it's supposed to).